Extension/src/LanguageServer/extension.ts

```diff
@@ -1003,6 +1003,16 @@ function handleCrashFileRead(err: NodeJS.ErrnoException, data: string): void {
         return logCrashTelemetry("readFile: " + err.code);
     }
 
+    // Extract the crashing process version, because the version might not match
+    // if multiple VS Codes are running with different extension versions.
+    let binaryVersion: string = "";
+    let startVersion: number = data.indexOf("Version:");
+    if (startVersion >= 0) {
+        data = data.substr(startVersion);
+        const binaryVersionMatches: string[] = data.match(/^Version:\s*(\d|\d*\.\d*\.\d*\.\d*)/);
+        binaryVersion = binaryVersionMatches && binaryVersionMatches.length > 1 ? binaryVersionMatches[1] : "";
```

Comment:
instead of "" do you want some diagnostic logging or "unknown"  ?

Reply:
Sure.

Reply:
Umm...actually, if the version is missing for some unexpected reason we'll see it in the crash logs itself, so having a separate telemetry message seems unnecessary.